main/lsp/notifications/did_change_configuration.cc

```diff
@@ -0,0 +1,22 @@
+#include "main/lsp/notifications/did_change_configuration.h"
+#include "main/lsp/LSPPreprocessor.h"
+
+namespace sorbet::realmain::lsp {
+DidChangeConfigurationTask::DidChangeConfigurationTask(const LSPConfiguration &config,
+                                                       std::unique_ptr<DidChangeConfigurationParams> params)
+    : LSPTask(config, LSPMethod::WorkspaceDidChangeConfiguration), params(move(params)) {}
+
+LSPTask::Phase DidChangeConfigurationTask::finalPhase() const {
+    return LSPTask::Phase::RUN;
```

Comment:
Can we add a comment here basically saying that we want our task to run all the way to type checker so that the update propagates to all globalstates that are alive in the LSP process?

Reply:
sure